data/seeds/001_profiles.js

```diff
@@ -22,56 +22,56 @@ const profiles = [
     email: 'Test001@maildrop.cc',
     name: 'Test001 User',
     okta_id: '00ulthapbErVUwVJy4x6',
-    role: 'super_admin',
+    role: 1,
```

Comment:
Should these be "role_id" instead of "role"

Reply:
Let me connect with Daniel, but I'm pretty sure this does need to be changed.